osu.Game/Screens/Play/Player.cs

```diff
@@ -333,5 +344,22 @@ private void dimChanged(object sender, EventArgs e)
         private Bindable<bool> mouseWheelDisabled;
 
         protected override bool OnWheel(InputState state) => mouseWheelDisabled.Value && !IsPaused;
+
+        protected override bool OnKeyDown(InputState state, KeyDownEventArgs args)
+        {
+            if (args.Repeat) return false;
+
+            if (state.Keyboard.ShiftPressed)
+            {
+                switch (args.Key)

```

Comment:
I don't think that you need a switch statement here. 
You're only using it for one comparison so you might want to use if.